aptos-move/framework/aptos-stdlib/sources/cryptography/pedersen.move

```diff
@@ -0,0 +1,167 @@
+/// This module implements a Pedersen commitment API, over the Ristretto255 curve, that can be used with the
+/// Bulletproofs module.
+///
+/// A Pedersen commitment to a value `v` under _commitment key_ `(g, h)` is `v * g + r * h`, for a random scalar `r`.
+
+module aptos_std::pedersen {
```

Comment:
should we be explicit and call it `pedersen_ristretto255`, in case we need a generic pedersen in the future?

Reply:
same q for elgamal module

Reply:
Fixing this.